browser/src/layer/marker/TextInput.js

```diff
@@ -159,7 +159,7 @@ L.TextInput = L.Layer.extend({
 	},
 
 	disable: function () {
-		this._textArea.setAttribute('disabled', true);
```

Comment:
please do not modify TextInput.js file, it's unrelated to JSDialog, its about keyboard handling